tests/strictequal.test.js

```diff
@@ -0,0 +1,52 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictEqual = require('../strictequal')
+
+describe('test 1 fail on parameter types types', () => {
+  describe('test 1 parameter are different types', () => {
+    it('test 1 types differ', () => {
+      // create test data for fail test
+      const firstValTest1 = '5'
+      const secondValTest1 = 5
+
+      //call the function
+      let test1Result = strictEqual(firstValTest1, secondValTest1)
+
+      //make assertion the test fails
+      expect(test1Result).to.equal(false)
+    })
+  })
+})
+
+describe('test 2 fail on parameter values', () => {
+  describe('test 2 parameter are different values', () => {
+    it('test 2 values differ', () => {
+      // create test data for fail test
+      const firstValTest1 = 6
+      const secondValTest1 = 5
+
+      //call the function
+      let test1Result = strictEqual(firstValTest1, secondValTest1)
```

Comment:
This should be `const` since you do not reassign a value to this variable. Also you have `test1` variables in your test2. As a rule, you would never number tests.